### PR TITLE
SSCSSI-337: Fix broken trigger for CcdCallbackMapService - Post Heari…

### PR DESCRIPTION
**Pull request by srochani**
…ng Review Event

### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/SSCSSI-337

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewSubmittedHandlerTest.java [hunk 1]
```diff
@@ -147,6 +157,46 @@ void givenActionTypeSetAsideSelectedIsNotRefuse_shouldNotUpdateCaseWIthCcdServic
...
 
+        verify(ccdCallbackMapService, never())
+                .handleCcdCallbackMapV2(eq(action), anyLong(), consumerArgumentCaptor.capture());
+
+        verifyNoInteractions(ccdService);
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = SetAsideActions.class, names = {"GRANT"})
+    void givenActionTypeSetAsideSelectedIsNotRefuse_shouldNotUpdateCaseWIthCcdService_whenCcdCallbackMapV2IsEnabled(SetAsideActions action) {
```
Comment (sanjayparekh-hmcts) [inline src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewSubmittedHandlerTest.java:168]: is there any benefit to having this a ParameterizedTest if there is only one enum value?

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewSubmittedHandlerTest.java [hunk 2]
```diff
@@ -183,6 +233,51 @@ void givenRefusedSetAsideSelected_andNoStatementOfReasons_shouldReturnCorrectCal
...
 
+        verify(ccdCallbackMapService, never())
+                .handleCcdCallbackMapV2(eq(REFUSE), anyLong(), consumerArgumentCaptor.capture());
+
+        assertThat(response.getData().getState()).isEqualTo(State.DORMANT_APPEAL_STATE);
+        assertThat(response.getData().getInterlocReviewState()).isEqualTo(InterlocReviewState.NONE);
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = YesNo.class, names = {"NO"})
```
Comment (sanjayparekh-hmcts) [inline src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewSubmittedHandlerTest.java:244]: same Q here

**Review by sanjayparekh-hmcts — CHANGES_REQUESTED** [review]

**Review by SomeshRA — APPROVED** [review]
LGTM

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Review by SomeshRA — APPROVED** [review]
LGTM

**Pull request merged**